Building from source error: command 'gcc' failed with exit status 1

Hi There,
I'm trying to install Pytorch as a module on my university's computing cluster. The node uses CentOS (6.4) x86_64.

I install running these commands:

    module load gcclib/5.2.0 cmake/3.8.2
    module load gcc/5.2.0
    module load anaconda3/4.0.0
    export CMAKE_PREFIX_PATH="$(dirname $(which conda))/../"
    export NO_CUDA=1
    git clone --recursive https://github.com/pytorch/pytorch
    cd pytorch/
    python setup.py install --prefix=${HOME}
    
Things look fine as it installs but then I run into the following errors, ending in the process terminating with error relating to GCC? I've tried with version GCC 6.2.0 and the same result occurs. Not sure what to even try to fix this!

```h/csrc/jit/passes/peephole.o -D_THP_CORE -std=c++11 -Wno-write-strings -fno-strict-aliasing -Wno-missing-braces -DWITH_NUMPY -DWITH_DISTRIBUTED
In file included from torch/csrc/THP.h:46:0,
                 from torch/csrc/Storage.cpp:12:
/bg01/homescinet/j/josselyn/username/pytorch/torch/csrc/generic/Storage.cpp: In function 'PyObject* THPFloatStorage_pynew(PyTypeObject*, PyObject*, PyObject*)':
/bg01/homescinet/j/josselyn/username/pytorch/torch/csrc/generic/Storage.cpp:114:13: error: expected ')' before 'PRId64'
         "%" PRId64 ", but the viewed storage has only %" PRId64 " element(s)", 
             ^
torch/csrc/utils.h:120:49: note: in definition of macro 'THPUtils_assertRet'
 if (THP_EXPECT(!(cond), 0)) { THPUtils_setError(__VA_ARGS__); return value; }
                                                 ^
/bg01/homescinet/j/josselyn/username/pytorch/torch/csrc/generic/Storage.cpp:113:5: note: in expansion of macro 'THPUtils_assert'
     THPUtils_assert(offset >= 0 && offset <= numel, "specified an offset of "
     ^
/bg01/homescinet/j/josselyn/username/pytorch/torch/csrc/generic/Storage.cpp:116:13: error: expected ')' before 'PRId64'
         "%" PRId64 ", but the viewed storage has only %" PRId64 " element(s) af
             ^
torch/csrc/utils.h:120:49: note: in definition of macro 'THPUtils_assertRet'
 if (THP_EXPECT(!(cond), 0)) { THPUtils_setError(__VA_ARGS__); return value; }
                                                 ^
/bg01/homescinet/j/josselyn/username/pytorch/torch/csrc/generic/Storage.cpp:115:5: note: in expansion of macro 'THPUtils_assert'
     THPUtils_assert(size >= 1 && size <= numel - offset, "specified a size of "
     ^
In file included from generic/Storage.cpp:1:0,
                 from /bg01/homescinet/j/josselyn/username/pytorch/torch/lib/tmp_install/include/TH/THGenerateFloatTypes.h:10,
                 from /bg01/homescinet/j/josselyn/username/pytorch/torch/lib/tmp_install/include/TH/THGenerateAllTypes.h:10,
                 from torch/csrc/Storage.cpp:16:
/bg01/homescinet/j/josselyn/username/pytorch/torch/csrc/generic/Storage.cpp: In function 'PyObject* THPFloatStorage_get(THPFloatStorage*, PyObject*)':
/bg01/homescinet/j/josselyn/username/pytorch/torch/csrc/generic/Storage.cpp:192:48: error: expected ')' before 'PRId64'
       PyErr_Format(PyExc_IndexError, "index %" PRId64 " out of range for storag
                                                ^
/bg01/homescinet/j/josselyn/username/pytorch/torch/csrc/generic/Storage.cpp:209:60: error: expected ')' before 'PRId64'
       THPUtils_setError("Trying to slice with a step of %" PRId64 ", but only a
                                                            ^
/bg01/homescinet/j/josselyn/username/pytorch/torch/csrc/generic/Storage.cpp: In function 'int THPFloatStorage_set(THPFloatStorage*, PyObject*, PyObject*)':
/bg01/homescinet/j/josselyn/username/pytorch/torch/csrc/generic/Storage.cpp:252:60: error: expected ')' before 'PRId64'
       THPUtils_setError("Trying to slice with a step of %" PRId64 ", but only a
                                                            ^
In file included from /bg01/homescinet/j/josselyn/username/pytorch/torch/csrc/generic/Storage.cpp:368:0,
                 from generic/Storage.cpp:1,
                 from /bg01/homescinet/j/josselyn/username/pytorch/torch/lib/tmp_install/include/TH/THGenerateFloatTypes.h:10,
                 from /bg01/homescinet/j/josselyn/username/pytorch/torch/lib/tmp_install/include/TH/THGenerateAllTypes.h:10,
                 from torch/csrc/Storage.cpp:16:
/bg01/homescinet/j/josselyn/username/pytorch/torch/csrc/generic/StorageMethods.cpp: In function 'PyObject* THPFloatStorage_fromBuffer(PyObject*, PyObject*, PyObject*)':
/bg01/homescinet/j/josselyn/username/pytorch/torch/csrc/generic/StorageMethods.cpp:130:74: error: expected ')' before 'PRId64'
  "offset must be non-negative and no greater than buffer length (%" PRId64 "), 
                                                                     ^
/bg01/homescinet/j/josselyn/username/pytorch/torch/csrc/generic/StorageMethods.cpp:138:55: error: expected ')' before 'PRId64'
       PyErr_Format(PyExc_ValueError, "buffer size (%" PRId64 ") must be a multi
                                                       ^
/bg01/homescinet/j/josselyn/username/pytorch/torch/csrc/generic/StorageMethods.cpp:147:56: error: expected ')' before 'PRId64'
     PyErr_Format(PyExc_ValueError, "buffer has only %" PRId64 " elements after 
                                                        ^
In file included from torch/csrc/THP.h:46:0,
                 from torch/csrc/Storage.cpp:12:
/bg01/homescinet/j/josselyn/username/pytorch/torch/csrc/generic/Storage.cpp: In function 'PyObject* THPDoubleStorage_pynew(PyTypeObject*, PyObject*, PyObject*)':
/bg01/homescinet/j/josselyn/username/pytorch/torch/csrc/generic/Storage.cpp:114:13: error: expected ')' before 'PRId64'
         "%" PRId64 ", but the viewed storage has only %" PRId64 " element(s)", 
             ^
torch/csrc/utils.h:120:49: note: in definition of macro 'THPUtils_assertRet'
 if (THP_EXPECT(!(cond), 0)) { THPUtils_setError(__VA_ARGS__); return value; }
                                                 ^
/bg01/homescinet/j/josselyn/username/pytorch/torch/csrc/generic/Storage.cpp:113:5: note: in expansion of macro 'THPUtils_assert'
     THPUtils_assert(offset >= 0 && offset <= numel, "specified an offset of "
     ^
/bg01/homescinet/j/josselyn/username/pytorch/torch/csrc/generic/Storage.cpp:116:13: error: expected ')' before 'PRId64'
         "%" PRId64 ", but the viewed storage has only %" PRId64 " element(s) af
             ^
torch/csrc/utils.h:120:49: note: in definition of macro 'THPUtils_assertRet'
 if (THP_EXPECT(!(cond), 0)) { THPUtils_setError(__VA_ARGS__); return value; }
                                                 ^
/bg01/homescinet/j/josselyn/username/pytorch/torch/csrc/generic/Storage.cpp:115:5: note: in expansion of macro 'THPUtils_assert'
     THPUtils_assert(size >= 1 && size <= numel - offset, "specified a size of "
     ^
In file included from generic/Storage.cpp:1:0,
                 from /bg01/homescinet/j/josselyn/username/pytorch/torch/lib/tmp_install/include/TH/THGenerateFloatTypes.h:11,
                 from /bg01/homescinet/j/josselyn/username/pytorch/torch/lib/tmp_install/include/TH/THGenerateAllTypes.h:10,
                 from torch/csrc/Storage.cpp:16:
/bg01/homescinet/j/josselyn/username/pytorch/torch/csrc/generic/Storage.cpp: In function 'PyObject* THPDoubleStorage_get(THPDoubleStorage*, PyObject*)':
/bg01/homescinet/j/josselyn/username/pytorch/torch/csrc/generic/Storage.cpp:192:48: error: expected ')' before 'PRId64'
       PyErr_Format(PyExc_IndexError, "index %" PRId64 " out of range for storag
                                                ^
/bg01/homescinet/j/josselyn/username/pytorch/torch/csrc/generic/Storage.cpp:209:60: error: expected ')' before 'PRId64'
       THPUtils_setError("Trying to slice with a step of %" PRId64 ", but only a
                                                            ^
/bg01/homescinet/j/josselyn/username/pytorch/torch/csrc/generic/Storage.cpp: In function 'int THPDoubleStorage_set(THPDoubleStorage*, PyObject*, PyObject*)':
/bg01/homescinet/j/josselyn/username/pytorch/torch/csrc/generic/Storage.cpp:252:60: error: expected ')' before 'PRId64'
       THPUtils_setError("Trying to slice with a step of %" PRId64 ", but only a
                                                            ^
In file included from /bg01/homescinet/j/josselyn/username/pytorch/torch/csrc/generic/Storage.cpp:368:0,
                 from generic/Storage.cpp:1,
                 from /bg01/homescinet/j/josselyn/username/pytorch/torch/lib/tmp_install/include/TH/THGenerateFloatTypes.h:11,
                 from /bg01/homescinet/j/josselyn/username/pytorch/torch/lib/tmp_install/include/TH/THGenerateAllTypes.h:10,
                 from torch/csrc/Storage.cpp:16:
/bg01/homescinet/j/josselyn/username/pytorch/torch/csrc/generic/StorageMethods.cpp: In function 'PyObject* THPDoubleStorage_fromBuffer(PyObject*, PyObject*, PyObject*)':
/bg01/homescinet/j/josselyn/username/pytorch/torch/csrc/generic/StorageMethods.cpp:130:74: error: expected ')' before 'PRId64'
  "offset must be non-negative and no greater than buffer length (%" PRId64 "), 
                                                                     ^
/bg01/homescinet/j/josselyn/username/pytorch/torch/csrc/generic/StorageMethods.cpp:138:55: error: expected ')' before 'PRId64'
       PyErr_Format(PyExc_ValueError, "buffer size (%" PRId64 ") must be a multi
                                                       ^
/bg01/homescinet/j/josselyn/username/pytorch/torch/csrc/generic/StorageMethods.cpp:147:56: error: expected ')' before 'PRId64'
     PyErr_Format(PyExc_ValueError, "buffer has only %" PRId64 " elements after 
                                                        ^
In file included from torch/csrc/THP.h:46:0,
                 from torch/csrc/Storage.cpp:12:
/bg01/homescinet/j/josselyn/username/pytorch/torch/csrc/generic/Storage.cpp: In function 'PyObject* THPByteStorage_pynew(PyTypeObject*, PyObject*, PyObject*)':
/bg01/homescinet/j/josselyn/username/pytorch/torch/csrc/generic/Storage.cpp:114:13: error: expected ')' before 'PRId64'
         "%" PRId64 ", but the viewed storage has only %" PRId64 " element(s)", 
             ^
torch/csrc/utils.h:120:49: note: in definition of macro 'THPUtils_assertRet'
 if (THP_EXPECT(!(cond), 0)) { THPUtils_setError(__VA_ARGS__); return value; }
                                                 ^
/bg01/homescinet/j/josselyn/username/pytorch/torch/csrc/generic/Storage.cpp:113:5: note: in expansion of macro 'THPUtils_assert'
     THPUtils_assert(offset >= 0 && offset <= numel, "specified an offset of "
     ^
/bg01/homescinet/j/josselyn/username/pytorch/torch/csrc/generic/Storage.cpp:116:13: error: expected ')' before 'PRId64'
         "%" PRId64 ", but the viewed storage has only %" PRId64 " element(s) af
             ^
torch/csrc/utils.h:120:49: note: in definition of macro 'THPUtils_assertRet'
 if (THP_EXPECT(!(cond), 0)) { THPUtils_setError(__VA_ARGS__); return value; }
                                                 ^
/bg01/homescinet/j/josselyn/username/pytorch/torch/csrc/generic/Storage.cpp:115:5: note: in expansion of macro 'THPUtils_assert'
     THPUtils_assert(size >= 1 && size <= numel - offset, "specified a size of "
     ^
In file included from generic/Storage.cpp:1:0,
                 from /bg01/homescinet/j/josselyn/username/pytorch/torch/lib/tmp_install/include/TH/THGenerateIntTypes.h:10,
                 from /bg01/homescinet/j/josselyn/username/pytorch/torch/lib/tmp_install/include/TH/THGenerateAllTypes.h:11,
                 from torch/csrc/Storage.cpp:16:
/bg01/homescinet/j/josselyn/username/pytorch/torch/csrc/generic/Storage.cpp: In function 'PyObject* THPByteStorage_get(THPByteStorage*, PyObject*)':
/bg01/homescinet/j/josselyn/username/pytorch/torch/csrc/generic/Storage.cpp:192:48: error: expected ')' before 'PRId64'
       PyErr_Format(PyExc_IndexError, "index %" PRId64 " out of range for storag
                                                ^
/bg01/homescinet/j/josselyn/username/pytorch/torch/csrc/generic/Storage.cpp:209:60: error: expected ')' before 'PRId64'
       THPUtils_setError("Trying to slice with a step of %" PRId64 ", but only a
                                                            ^
/bg01/homescinet/j/josselyn/username/pytorch/torch/csrc/generic/Storage.cpp: In function 'int THPByteStorage_set(THPByteStorage*, PyObject*, PyObject*)':
/bg01/homescinet/j/josselyn/username/pytorch/torch/csrc/generic/Storage.cpp:252:60: error: expected ')' before 'PRId64'
       THPUtils_setError("Trying to slice with a step of %" PRId64 ", but only a
                                                            ^
In file included from /bg01/homescinet/j/josselyn/username/pytorch/torch/csrc/generic/Storage.cpp:368:0,
                 from generic/Storage.cpp:1,
                 from /bg01/homescinet/j/josselyn/username/pytorch/torch/lib/tmp_install/include/TH/THGenerateIntTypes.h:10,
                 from /bg01/homescinet/j/josselyn/username/pytorch/torch/lib/tmp_install/include/TH/THGenerateAllTypes.h:11,
                 from torch/csrc/Storage.cpp:16:
/bg01/homescinet/j/josselyn/username/pytorch/torch/csrc/generic/StorageMethods.cpp: In function 'PyObject* THPByteStorage_fromBuffer(PyObject*, PyObject*, PyObject*)':
/bg01/homescinet/j/josselyn/username/pytorch/torch/csrc/generic/StorageMethods.cpp:130:74: error: expected ')' before 'PRId64'
  "offset must be non-negative and no greater than buffer length (%" PRId64 "), 
                                                                     ^
/bg01/homescinet/j/josselyn/username/pytorch/torch/csrc/generic/StorageMethods.cpp:138:55: error: expected ')' before 'PRId64'
       PyErr_Format(PyExc_ValueError, "buffer size (%" PRId64 ") must be a multi
                                                       ^
/bg01/homescinet/j/josselyn/username/pytorch/torch/csrc/generic/StorageMethods.cpp:147:56: error: expected ')' before 'PRId64'
     PyErr_Format(PyExc_ValueError, "buffer has only %" PRId64 " elements after 
                                                        ^
In file included from torch/csrc/THP.h:46:0,
                 from torch/csrc/Storage.cpp:12:
/bg01/homescinet/j/josselyn/username/pytorch/torch/csrc/generic/Storage.cpp: In function 'PyObject* THPCharStorage_pynew(PyTypeObject*, PyObject*, PyObject*)':
/bg01/homescinet/j/josselyn/username/pytorch/torch/csrc/generic/Storage.cpp:114:13: error: expected ')' before 'PRId64'
         "%" PRId64 ", but the viewed storage has only %" PRId64 " element(s)", 
             ^
torch/csrc/utils.h:120:49: note: in definition of macro 'THPUtils_assertRet'
 if (THP_EXPECT(!(cond), 0)) { THPUtils_setError(__VA_ARGS__); return value; }
                                                 ^
/bg01/homescinet/j/josselyn/username/pytorch/torch/csrc/generic/Storage.cpp:113:5: note: in expansion of macro 'THPUtils_assert'
     THPUtils_assert(offset >= 0 && offset <= numel, "specified an offset of "
     ^
/bg01/homescinet/j/josselyn/username/pytorch/torch/csrc/generic/Storage.cpp:116:13: error: expected ')' before 'PRId64'
         "%" PRId64 ", but the viewed storage has only %" PRId64 " element(s) af
             ^
torch/csrc/utils.h:120:49: note: in definition of macro 'THPUtils_assertRet'
 if (THP_EXPECT(!(cond), 0)) { THPUtils_setError(__VA_ARGS__); return value; }
                                                 ^
/bg01/homescinet/j/josselyn/username/pytorch/torch/csrc/generic/Storage.cpp:115:5: note: in expansion of macro 'THPUtils_assert'
     THPUtils_assert(size >= 1 && size <= numel - offset, "specified a size of "
     ^
In file included from generic/Storage.cpp:1:0,
                 from /bg01/homescinet/j/josselyn/username/pytorch/torch/lib/tmp_install/include/TH/THGenerateIntTypes.h:11,
                 from /bg01/homescinet/j/josselyn/username/pytorch/torch/lib/tmp_install/include/TH/THGenerateAllTypes.h:11,
                 from torch/csrc/Storage.cpp:16:
/bg01/homescinet/j/josselyn/username/pytorch/torch/csrc/generic/Storage.cpp: In function 'PyObject* THPCharStorage_get(THPCharStorage*, PyObject*)':
/bg01/homescinet/j/josselyn/username/pytorch/torch/csrc/generic/Storage.cpp:192:48: error: expected ')' before 'PRId64'
       PyErr_Format(PyExc_IndexError, "index %" PRId64 " out of range for storag
                                                ^
/bg01/homescinet/j/josselyn/username/pytorch/torch/csrc/generic/Storage.cpp:209:60: error: expected ')' before 'PRId64'
       THPUtils_setError("Trying to slice with a step of %" PRId64 ", but only a
                                                            ^
/bg01/homescinet/j/josselyn/username/pytorch/torch/csrc/generic/Storage.cpp: In function 'int THPCharStorage_set(THPCharStorage*, PyObject*, PyObject*)':
/bg01/homescinet/j/josselyn/username/pytorch/torch/csrc/generic/Storage.cpp:252:60: error: expected ')' before 'PRId64'
       THPUtils_setError("Trying to slice with a step of %" PRId64 ", but only a
                                                            ^
In file included from /bg01/homescinet/j/josselyn/username/pytorch/torch/csrc/generic/Storage.cpp:368:0,
                 from generic/Storage.cpp:1,
                 from /bg01/homescinet/j/josselyn/username/pytorch/torch/lib/tmp_install/include/TH/THGenerateIntTypes.h:11,
                 from /bg01/homescinet/j/josselyn/username/pytorch/torch/lib/tmp_install/include/TH/THGenerateAllTypes.h:11,
                 from torch/csrc/Storage.cpp:16:
/bg01/homescinet/j/josselyn/username/pytorch/torch/csrc/generic/StorageMethods.cpp: In function 'PyObject* THPCharStorage_fromBuffer(PyObject*, PyObject*, PyObject*)':
/bg01/homescinet/j/josselyn/username/pytorch/torch/csrc/generic/StorageMethods.cpp:130:74: error: expected ')' before 'PRId64'
  "offset must be non-negative and no greater than buffer length (%" PRId64 "), 
                                                                     ^
/bg01/homescinet/j/josselyn/username/pytorch/torch/csrc/generic/StorageMethods.cpp:138:55: error: expected ')' before 'PRId64'
       PyErr_Format(PyExc_ValueError, "buffer size (%" PRId64 ") must be a multi
                                                       ^
/bg01/homescinet/j/josselyn/username/pytorch/torch/csrc/generic/StorageMethods.cpp:147:56: error: expected ')' before 'PRId64'
     PyErr_Format(PyExc_ValueError, "buffer has only %" PRId64 " elements after 
                                                        ^
In file included from torch/csrc/THP.h:46:0,
                 from torch/csrc/Storage.cpp:12:
/bg01/homescinet/j/josselyn/username/pytorch/torch/csrc/generic/Storage.cpp: In function 'PyObject* THPShortStorage_pynew(PyTypeObject*, PyObject*, PyObject*)':
/bg01/homescinet/j/josselyn/username/pytorch/torch/csrc/generic/Storage.cpp:114:13: error: expected ')' before 'PRId64'
         "%" PRId64 ", but the viewed storage has only %" PRId64 " element(s)", 
             ^
torch/csrc/utils.h:120:49: note: in definition of macro 'THPUtils_assertRet'
 if (THP_EXPECT(!(cond), 0)) { THPUtils_setError(__VA_ARGS__); return value; }
                                                 ^
/bg01/homescinet/j/josselyn/username/pytorch/torch/csrc/generic/Storage.cpp:113:5: note: in expansion of macro 'THPUtils_assert'
     THPUtils_assert(offset >= 0 && offset <= numel, "specified an offset of "
     ^
/bg01/homescinet/j/josselyn/username/pytorch/torch/csrc/generic/Storage.cpp:116:13: error: expected ')' before 'PRId64'
         "%" PRId64 ", but the viewed storage has only %" PRId64 " element(s) af
             ^
torch/csrc/utils.h:120:49: note: in definition of macro 'THPUtils_assertRet'
 if (THP_EXPECT(!(cond), 0)) { THPUtils_setError(__VA_ARGS__); return value; }
                                                 ^
/bg01/homescinet/j/josselyn/username/pytorch/torch/csrc/generic/Storage.cpp:115:5: note: in expansion of macro 'THPUtils_assert'
     THPUtils_assert(size >= 1 && size <= numel - offset, "specified a size of "
     ^
In file included from generic/Storage.cpp:1:0,
                 from /bg01/homescinet/j/josselyn/username/pytorch/torch/lib/tmp_install/include/TH/THGenerateIntTypes.h:12,
                 from /bg01/homescinet/j/josselyn/username/pytorch/torch/lib/tmp_install/include/TH/THGenerateAllTypes.h:11,
                 from torch/csrc/Storage.cpp:16:
/bg01/homescinet/j/josselyn/username/pytorch/torch/csrc/generic/Storage.cpp: In function 'PyObject* THPShortStorage_get(THPShortStorage*, PyObject*)':
/bg01/homescinet/j/josselyn/username/pytorch/torch/csrc/generic/Storage.cpp:192:48: error: expected ')' before 'PRId64'
       PyErr_Format(PyExc_IndexError, "index %" PRId64 " out of range for storag
                                                ^
/bg01/homescinet/j/josselyn/username/pytorch/torch/csrc/generic/Storage.cpp:209:60: error: expected ')' before 'PRId64'
       THPUtils_setError("Trying to slice with a step of %" PRId64 ", but only a
                                                            ^
/bg01/homescinet/j/josselyn/username/pytorch/torch/csrc/generic/Storage.cpp: In function 'int THPShortStorage_set(THPShortStorage*, PyObject*, PyObject*)':
/bg01/homescinet/j/josselyn/username/pytorch/torch/csrc/generic/Storage.cpp:252:60: error: expected ')' before 'PRId64'
       THPUtils_setError("Trying to slice with a step of %" PRId64 ", but only a
                                                            ^
In file included from /bg01/homescinet/j/josselyn/username/pytorch/torch/csrc/generic/Storage.cpp:368:0,
                 from generic/Storage.cpp:1,
                 from /bg01/homescinet/j/josselyn/username/pytorch/torch/lib/tmp_install/include/TH/THGenerateIntTypes.h:12,
                 from /bg01/homescinet/j/josselyn/username/pytorch/torch/lib/tmp_install/include/TH/THGenerateAllTypes.h:11,
                 from torch/csrc/Storage.cpp:16:
/bg01/homescinet/j/josselyn/username/pytorch/torch/csrc/generic/StorageMethods.cpp: In function 'PyObject* THPShortStorage_fromBuffer(PyObject*, PyObject*, PyObject*)':
/bg01/homescinet/j/josselyn/username/pytorch/torch/csrc/generic/StorageMethods.cpp:130:74: error: expected ')' before 'PRId64'
  "offset must be non-negative and no greater than buffer length (%" PRId64 "), 
                                                                     ^
/bg01/homescinet/j/josselyn/username/pytorch/torch/csrc/generic/StorageMethods.cpp:138:55: error: expected ')' before 'PRId64'
       PyErr_Format(PyExc_ValueError, "buffer size (%" PRId64 ") must be a multi
                                                       ^
/bg01/homescinet/j/josselyn/username/pytorch/torch/csrc/generic/StorageMethods.cpp:147:56: error: expected ')' before 'PRId64'
     PyErr_Format(PyExc_ValueError, "buffer has only %" PRId64 " elements after 
                                                        ^
In file included from torch/csrc/THP.h:46:0,
                 from torch/csrc/Storage.cpp:12:
/bg01/homescinet/j/josselyn/username/pytorch/torch/csrc/generic/Storage.cpp: In function 'PyObject* THPIntStorage_pynew(PyTypeObject*, PyObject*, PyObject*)':
/bg01/homescinet/j/josselyn/username/pytorch/torch/csrc/generic/Storage.cpp:114:13: error: expected ')' before 'PRId64'
         "%" PRId64 ", but the viewed storage has only %" PRId64 " element(s)", 
             ^
torch/csrc/utils.h:120:49: note: in definition of macro 'THPUtils_assertRet'
 if (THP_EXPECT(!(cond), 0)) { THPUtils_setError(__VA_ARGS__); return value; }
                                                 ^
/bg01/homescinet/j/josselyn/username/pytorch/torch/csrc/generic/Storage.cpp:113:5: note: in expansion of macro 'THPUtils_assert'
     THPUtils_assert(offset >= 0 && offset <= numel, "specified an offset of "
     ^
/bg01/homescinet/j/josselyn/username/pytorch/torch/csrc/generic/Storage.cpp:116:13: error: expected ')' before 'PRId64'
         "%" PRId64 ", but the viewed storage has only %" PRId64 " element(s) af
             ^
torch/csrc/utils.h:120:49: note: in definition of macro 'THPUtils_assertRet'
 if (THP_EXPECT(!(cond), 0)) { THPUtils_setError(__VA_ARGS__); return value; }
                                                 ^
/bg01/homescinet/j/josselyn/username/pytorch/torch/csrc/generic/Storage.cpp:115:5: note: in expansion of macro 'THPUtils_assert'
     THPUtils_assert(size >= 1 && size <= numel - offset, "specified a size of "
     ^
In file included from generic/Storage.cpp:1:0,
                 from /bg01/homescinet/j/josselyn/username/pytorch/torch/lib/tmp_install/include/TH/THGenerateIntTypes.h:13,
                 from /bg01/homescinet/j/josselyn/username/pytorch/torch/lib/tmp_install/include/TH/THGenerateAllTypes.h:11,
                 from torch/csrc/Storage.cpp:16:
/bg01/homescinet/j/josselyn/username/pytorch/torch/csrc/generic/Storage.cpp: In function 'PyObject* THPIntStorage_get(THPIntStorage*, PyObject*)':
/bg01/homescinet/j/josselyn/username/pytorch/torch/csrc/generic/Storage.cpp:192:48: error: expected ')' before 'PRId64'
       PyErr_Format(PyExc_IndexError, "index %" PRId64 " out of range for storag
                                                ^
/bg01/homescinet/j/josselyn/username/pytorch/torch/csrc/generic/Storage.cpp:209:60: error: expected ')' before 'PRId64'
       THPUtils_setError("Trying to slice with a step of %" PRId64 ", but only a
                                                            ^
/bg01/homescinet/j/josselyn/username/pytorch/torch/csrc/generic/Storage.cpp: In function 'int THPIntStorage_set(THPIntStorage*, PyObject*, PyObject*)':
/bg01/homescinet/j/josselyn/username/pytorch/torch/csrc/generic/Storage.cpp:252:60: error: expected ')' before 'PRId64'
       THPUtils_setError("Trying to slice with a step of %" PRId64 ", but only a
                                                            ^
In file included from /bg01/homescinet/j/josselyn/username/pytorch/torch/csrc/generic/Storage.cpp:368:0,
                 from generic/Storage.cpp:1,
                 from /bg01/homescinet/j/josselyn/username/pytorch/torch/lib/tmp_install/include/TH/THGenerateIntTypes.h:13,
                 from /bg01/homescinet/j/josselyn/username/pytorch/torch/lib/tmp_install/include/TH/THGenerateAllTypes.h:11,
                 from torch/csrc/Storage.cpp:16:
/bg01/homescinet/j/josselyn/username/pytorch/torch/csrc/generic/StorageMethods.cpp: In function 'PyObject* THPIntStorage_fromBuffer(PyObject*, PyObject*, PyObject*)':
/bg01/homescinet/j/josselyn/username/pytorch/torch/csrc/generic/StorageMethods.cpp:130:74: error: expected ')' before 'PRId64'
  "offset must be non-negative and no greater than buffer length (%" PRId64 "), 
                                                                     ^
/bg01/homescinet/j/josselyn/username/pytorch/torch/csrc/generic/StorageMethods.cpp:138:55: error: expected ')' before 'PRId64'
       PyErr_Format(PyExc_ValueError, "buffer size (%" PRId64 ") must be a multi
                                                       ^
/bg01/homescinet/j/josselyn/username/pytorch/torch/csrc/generic/StorageMethods.cpp:147:56: error: expected ')' before 'PRId64'
     PyErr_Format(PyExc_ValueError, "buffer has only %" PRId64 " elements after 
                                                        ^
In file included from torch/csrc/THP.h:46:0,
                 from torch/csrc/Storage.cpp:12:
/bg01/homescinet/j/josselyn/username/pytorch/torch/csrc/generic/Storage.cpp: In function 'PyObject* THPLongStorage_pynew(PyTypeObject*, PyObject*, PyObject*)':
/bg01/homescinet/j/josselyn/username/pytorch/torch/csrc/generic/Storage.cpp:114:13: error: expected ')' before 'PRId64'
         "%" PRId64 ", but the viewed storage has only %" PRId64 " element(s)", 
             ^
torch/csrc/utils.h:120:49: note: in definition of macro 'THPUtils_assertRet'
 if (THP_EXPECT(!(cond), 0)) { THPUtils_setError(__VA_ARGS__); return value; }
                                                 ^
/bg01/homescinet/j/josselyn/username/pytorch/torch/csrc/generic/Storage.cpp:113:5: note: in expansion of macro 'THPUtils_assert'
     THPUtils_assert(offset >= 0 && offset <= numel, "specified an offset of "
     ^
/bg01/homescinet/j/josselyn/username/pytorch/torch/csrc/generic/Storage.cpp:116:13: error: expected ')' before 'PRId64'
         "%" PRId64 ", but the viewed storage has only %" PRId64 " element(s) af
             ^
torch/csrc/utils.h:120:49: note: in definition of macro 'THPUtils_assertRet'
 if (THP_EXPECT(!(cond), 0)) { THPUtils_setError(__VA_ARGS__); return value; }
                                                 ^
/bg01/homescinet/j/josselyn/username/pytorch/torch/csrc/generic/Storage.cpp:115:5: note: in expansion of macro 'THPUtils_assert'
     THPUtils_assert(size >= 1 && size <= numel - offset, "specified a size of "
     ^
In file included from generic/Storage.cpp:1:0,
                 from /bg01/homescinet/j/josselyn/username/pytorch/torch/lib/tmp_install/include/TH/THGenerateIntTypes.h:14,
                 from /bg01/homescinet/j/josselyn/username/pytorch/torch/lib/tmp_install/include/TH/THGenerateAllTypes.h:11,
                 from torch/csrc/Storage.cpp:16:
/bg01/homescinet/j/josselyn/username/pytorch/torch/csrc/generic/Storage.cpp: In function 'PyObject* THPLongStorage_get(THPLongStorage*, PyObject*)':
/bg01/homescinet/j/josselyn/username/pytorch/torch/csrc/generic/Storage.cpp:192:48: error: expected ')' before 'PRId64'
       PyErr_Format(PyExc_IndexError, "index %" PRId64 " out of range for storag
                                                ^
/bg01/homescinet/j/josselyn/username/pytorch/torch/csrc/generic/Storage.cpp:209:60: error: expected ')' before 'PRId64'
       THPUtils_setError("Trying to slice with a step of %" PRId64 ", but only a
                                                            ^
/bg01/homescinet/j/josselyn/username/pytorch/torch/csrc/generic/Storage.cpp: In function 'int THPLongStorage_set(THPLongStorage*, PyObject*, PyObject*)':
/bg01/homescinet/j/josselyn/username/pytorch/torch/csrc/generic/Storage.cpp:252:60: error: expected ')' before 'PRId64'
       THPUtils_setError("Trying to slice with a step of %" PRId64 ", but only a
                                                            ^
In file included from /bg01/homescinet/j/josselyn/username/pytorch/torch/csrc/generic/Storage.cpp:368:0,
                 from generic/Storage.cpp:1,
                 from /bg01/homescinet/j/josselyn/username/pytorch/torch/lib/tmp_install/include/TH/THGenerateIntTypes.h:14,
                 from /bg01/homescinet/j/josselyn/username/pytorch/torch/lib/tmp_install/include/TH/THGenerateAllTypes.h:11,
                 from torch/csrc/Storage.cpp:16:
/bg01/homescinet/j/josselyn/username/pytorch/torch/csrc/generic/StorageMethods.cpp: In function 'PyObject* THPLongStorage_fromBuffer(PyObject*, PyObject*, PyObject*)':
/bg01/homescinet/j/josselyn/username/pytorch/torch/csrc/generic/StorageMethods.cpp:130:74: error: expected ')' before 'PRId64'
  "offset must be non-negative and no greater than buffer length (%" PRId64 "), 
                                                                     ^
/bg01/homescinet/j/josselyn/username/pytorch/torch/csrc/generic/StorageMethods.cpp:138:55: error: expected ')' before 'PRId64'
       PyErr_Format(PyExc_ValueError, "buffer size (%" PRId64 ") must be a multi
                                                       ^
/bg01/homescinet/j/josselyn/username/pytorch/torch/csrc/generic/StorageMethods.cpp:147:56: error: expected ')' before 'PRId64'
     PyErr_Format(PyExc_ValueError, "buffer has only %" PRId64 " elements after 
                                                        ^
In file included from torch/csrc/THP.h:46:0,
                 from torch/csrc/Storage.cpp:12:
/bg01/homescinet/j/josselyn/username/pytorch/torch/csrc/generic/Storage.cpp: In function 'PyObject* THPHalfStorage_pynew(PyTypeObject*, PyObject*, PyObject*)':
/bg01/homescinet/j/josselyn/username/pytorch/torch/csrc/generic/Storage.cpp:114:13: error: expected ')' before 'PRId64'
         "%" PRId64 ", but the viewed storage has only %" PRId64 " element(s)", 
             ^
torch/csrc/utils.h:120:49: note: in definition of macro 'THPUtils_assertRet'
 if (THP_EXPECT(!(cond), 0)) { THPUtils_setError(__VA_ARGS__); return value; }
                                                 ^
/bg01/homescinet/j/josselyn/username/pytorch/torch/csrc/generic/Storage.cpp:113:5: note: in expansion of macro 'THPUtils_assert'
     THPUtils_assert(offset >= 0 && offset <= numel, "specified an offset of "
     ^
/bg01/homescinet/j/josselyn/username/pytorch/torch/csrc/generic/Storage.cpp:116:13: error: expected ')' before 'PRId64'
         "%" PRId64 ", but the viewed storage has only %" PRId64 " element(s) af
             ^
torch/csrc/utils.h:120:49: note: in definition of macro 'THPUtils_assertRet'
 if (THP_EXPECT(!(cond), 0)) { THPUtils_setError(__VA_ARGS__); return value; }
                                                 ^
/bg01/homescinet/j/josselyn/username/pytorch/torch/csrc/generic/Storage.cpp:115:5: note: in expansion of macro 'THPUtils_assert'
     THPUtils_assert(size >= 1 && size <= numel - offset, "specified a size of "
     ^
In file included from generic/Storage.cpp:1:0,
                 from torch/csrc/Storage.cpp:19:
/bg01/homescinet/j/josselyn/username/pytorch/torch/csrc/generic/Storage.cpp: In function 'PyObject* THPHalfStorage_get(THPHalfStorage*, PyObject*)':
/bg01/homescinet/j/josselyn/username/pytorch/torch/csrc/generic/Storage.cpp:192:48: error: expected ')' before 'PRId64'
       PyErr_Format(PyExc_IndexError, "index %" PRId64 " out of range for storag
                                                ^
/bg01/homescinet/j/josselyn/username/pytorch/torch/csrc/generic/Storage.cpp:209:60: error: expected ')' before 'PRId64'
       THPUtils_setError("Trying to slice with a step of %" PRId64 ", but only a
                                                            ^
/bg01/homescinet/j/josselyn/username/pytorch/torch/csrc/generic/Storage.cpp: In function 'int THPHalfStorage_set(THPHalfStorage*, PyObject*, PyObject*)':
/bg01/homescinet/j/josselyn/username/pytorch/torch/csrc/generic/Storage.cpp:252:60: error: expected ')' before 'PRId64'
       THPUtils_setError("Trying to slice with a step of %" PRId64 ", but only a
                                                            ^
gcc -Wsign-compare -Wunreachable-code -DNDEBUG -g -fwrapv -O3 -Wall -fPIC -I/bg01/homescinet/j/josselyn/username/pytorch -I/bg01/homescinet/j/josselyn/username/pytorch/torch/csrc -I/bg01/homescinet/j/josselyn/username/pytorch/torch/lib/pybind11/include -I/bg01/homescinet/j/josselyn/username/pytorch/torch/lib/tmp_install/include -I/bg01/homescinet/j/josselyn/username/pytorch/torch/lib/tmp_install/include/TH -I/bg01/homescinet/j/josselyn/username/pytorch/torch/lib/tmp_install/include/THNN -I/bg01/homescinet/j/josselyn/username/pytorch/torch/lib/tmp_install/include/ATen -I/scinet/gpc/Applications/anaconda3/4.0.0/lib/python3.5/site-packages/numpy/core/include -I/bg01/homescinet/j/josselyn/username/pytorch/torch/lib/tmp_install/include/THD -I/scinet/gpc/Applications/anaconda3/4.0.0/include/python3.5m -c torch/csrc/autograd/python_function.cpp -o build/temp.linux-x86_64-3.5/torch/csrc/autograd/python_function.o -D_THP_CORE -std=c++11 -Wno-write-strings -fno-strict-aliasing -Wno-missing-braces -DWITH_NUMPY -DWITH_DISTRIBUTED
In file included from /bg01/homescinet/j/josselyn/username/pytorch/torch/csrc/generic/Storage.cpp:368:0,
                 from generic/Storage.cpp:1,
                 from torch/csrc/Storage.cpp:19:
/bg01/homescinet/j/josselyn/username/pytorch/torch/csrc/generic/StorageMethods.cpp: In function 'PyObject* THPHalfStorage_fromBuffer(PyObject*, PyObject*, PyObject*)':
/bg01/homescinet/j/josselyn/username/pytorch/torch/csrc/generic/StorageMethods.cpp:130:74: error: expected ')' before 'PRId64'
  "offset must be non-negative and no greater than buffer length (%" PRId64 "), 
                                                                     ^
/bg01/homescinet/j/josselyn/username/pytorch/torch/csrc/generic/StorageMethods.cpp:138:55: error: expected ')' before 'PRId64'
       PyErr_Format(PyExc_ValueError, "buffer size (%" PRId64 ") must be a multi
                                                       ^
/bg01/homescinet/j/josselyn/username/pytorch/torch/csrc/generic/StorageMethods.cpp:147:56: error: expected ')' before 'PRId64'
     PyErr_Format(PyExc_ValueError, "buffer has only %" PRId64 " elements after 
                                                        ^
gcc -Wsign-compare -Wunreachable-code -DNDEBUG -g -fwrapv -O3 -Wall -fPIC -I/bg01/homescinet/j/josselyn/username/pytorch -I/bg01/homescinet/j/josselyn/username/pytorch/torch/csrc -I/bg01/homescinet/j/josselyn/username/pytorch/torch/lib/pybind11/include -I/bg01/homescinet/j/josselyn/username/pytorch/torch/lib/tmp_install/include -I/bg01/homescinet/j/josselyn/username/pytorch/torch/lib/tmp_install/include/TH -I/bg01/homescinet/j/josselyn/username/pytorch/torch/lib/tmp_install/include/THNN -I/bg01/homescinet/j/josselyn/username/pytorch/torch/lib/tmp_install/include/ATen -I/scinet/gpc/Applications/anaconda3/4.0.0/lib/python3.5/site-packages/numpy/core/include -I/bg01/homescinet/j/josselyn/username/pytorch/torch/lib/tmp_install/include/THD -I/scinet/gpc/Applications/anaconda3/4.0.0/include/python3.5m -c torch/csrc/autograd/python_variable.cpp -o build/temp.linux-x86_64-3.5/torch/csrc/autograd/python_variable.o -D_THP_CORE -std=c++11 -Wno-write-strings -fno-strict-aliasing -Wno-missing-braces -DWITH_NUMPY -DWITH_DISTRIBUTED
gcc -Wsign-compare -Wunreachable-code -DNDEBUG -g -fwrapv -O3 -Wall -fPIC -I/bg01/homescinet/j/josselyn/username/pytorch -I/bg01/homescinet/j/josselyn/username/pytorch/torch/csrc -I/bg01/homescinet/j/josselyn/username/pytorch/torch/lib/pybind11/include -I/bg01/homescinet/j/josselyn/username/pytorch/torch/lib/tmp_install/include -I/bg01/homescinet/j/josselyn/username/pytorch/torch/lib/tmp_install/include/TH -I/bg01/homescinet/j/josselyn/username/pytorch/torch/lib/tmp_install/include/THNN -I/bg01/homescinet/j/josselyn/username/pytorch/torch/lib/tmp_install/include/ATen -I/scinet/gpc/Applications/anaconda3/4.0.0/lib/python3.5/site-packages/numpy/core/include -I/bg01/homescinet/j/josselyn/username/pytorch/torch/lib/tmp_install/include/THD -I/scinet/gpc/Applications/anaconda3/4.0.0/include/python3.5m -c torch/csrc/jit/passes/onnx/peephole.cpp -o build/temp.linux-x86_64-3.5/torch/csrc/jit/passes/onnx/peephole.o -D_THP_CORE -std=c++11 -Wno-write-strings -fno-strict-aliasing -Wno-missing-braces -DWITH_NUMPY -DWITH_DISTRIBUTED
gcc -Wsign-compare -Wunreachable-code -DNDEBUG -g -fwrapv -O3 -Wall -fPIC -I/bg01/homescinet/j/josselyn/username/pytorch -I/bg01/homescinet/j/josselyn/username/pytorch/torch/csrc -I/bg01/homescinet/j/josselyn/username/pytorch/torch/lib/pybind11/include -I/bg01/homescinet/j/josselyn/username/pytorch/torch/lib/tmp_install/include -I/bg01/homescinet/j/josselyn/username/pytorch/torch/lib/tmp_install/include/TH -I/bg01/homescinet/j/josselyn/username/pytorch/torch/lib/tmp_install/include/THNN -I/bg01/homescinet/j/josselyn/username/pytorch/torch/lib/tmp_install/include/ATen -I/scinet/gpc/Applications/anaconda3/4.0.0/lib/python3.5/site-packages/numpy/core/include -I/bg01/homescinet/j/josselyn/username/pytorch/torch/lib/tmp_install/include/THD -I/scinet/gpc/Applications/anaconda3/4.0.0/include/python3.5m -c torch/csrc/jit/python_arg_flatten.cpp -o build/temp.linux-x86_64-3.5/torch/csrc/jit/python_arg_flatten.o -D_THP_CORE -std=c++11 -Wno-write-strings -fno-strict-aliasing -Wno-missing-braces -DWITH_NUMPY -DWITH_DISTRIBUTED
gcc -Wsign-compare -Wunreachable-code -DNDEBUG -g -fwrapv -O3 -Wall -fPIC -I/bg01/homescinet/j/josselyn/username/pytorch -I/bg01/homescinet/j/josselyn/username/pytorch/torch/csrc -I/bg01/homescinet/j/josselyn/username/pytorch/torch/lib/pybind11/include -I/bg01/homescinet/j/josselyn/username/pytorch/torch/lib/tmp_install/include -I/bg01/homescinet/j/josselyn/username/pytorch/torch/lib/tmp_install/include/TH -I/bg01/homescinet/j/josselyn/username/pytorch/torch/lib/tmp_install/include/THNN -I/bg01/homescinet/j/josselyn/username/pytorch/torch/lib/tmp_install/include/ATen -I/scinet/gpc/Applications/anaconda3/4.0.0/lib/python3.5/site-packages/numpy/core/include -I/bg01/homescinet/j/josselyn/username/pytorch/torch/lib/tmp_install/include/THD -I/scinet/gpc/Applications/anaconda3/4.0.0/include/python3.5m -c torch/csrc/autograd/python_hook.cpp -o build/temp.linux-x86_64-3.5/torch/csrc/autograd/python_hook.o -D_THP_CORE -std=c++11 -Wno-write-strings -fno-strict-aliasing -Wno-missing-braces -DWITH_NUMPY -DWITH_DISTRIBUTED
error: command 'gcc' failed with exit status 1
```

Thanks for any help you can provide!

--- a/torch/csrc/Storage.cpp
+++ b/torch/csrc/Storage.cpp
@@ -1,3 +1,5 @@
+#define __STDC_FORMAT_MACROS
+
 #include <Python.h>
 #ifdef _MSC_VER
 #include <Windows.h>

--- a/torch/csrc/Tensor.cpp
+++ b/torch/csrc/Tensor.cpp
@@ -1,3 +1,5 @@
+#define __STDC_FORMAT_MACROS
+
 #include <Python.h>
 #include <structmember.h>
 

--- a/torch/csrc/cuda/Storage.cpp
+++ b/torch/csrc/cuda/Storage.cpp
@@ -1,3 +1,5 @@
+#define __STDC_FORMAT_MACROS
+
 #include <Python.h>
 #include <structmember.h>
 

--- a/torch/csrc/cuda/Tensor.cpp
+++ b/torch/csrc/cuda/Tensor.cpp
@@ -1,3 +1,5 @@
+#define __STDC_FORMAT_MACROS
+
 #include <Python.h>
 #include <structmember.h>
 

